### ISSUE TITLE
Feature Request: A way to disable animations

### ISSUE THREAD

**Issue by zeGolem** [WayfireWM/wayfire#1763]
I couldn't find a config item to do so, and would like to be able to disable animations. While I appreciate nice and smooth animation like wayfire has, for usability reasons due to visual processing issues, I'd like to disable them. Mostly the window maximize/unmaximize, fast switcher, and workspace switcher animations

I was already able to remove some animations by setting the animation delay to 0, but those listed above don't seem to have such options…

It'd be nice to have a global toggle for it, that could also be used in a “power saving” mode :)

**Comment by ammen99** [comment]
> window maximize/unmaximize

This is controlled in the grid plugin, I'm pretty sure it has an option to do this.



> fast switcher

Fast switcher doesn't have any animations?


> workspace switcher animations

Vswitch also has an option for this: https://github.com/WayfireWM/wayfire/wiki/Configuration#vswitchduration

About having a centralized option, I can see why it can be useful, however it would require some changes in wf-config (but it won't be a breaking change in any case). It may be added in the future ;)

**Comment by zeGolem** [comment]
> This is controlled in the grid plugin, I'm pretty sure it has an option to do this.

Indeed! Sorry, I thought this was handled by wayfire itself and not a plugin

> Fast switcher doesn't have any animations?

Whoops, seems I was using the regular switcher rather than fast switcher, which is probably why I couldn't turn off the animations…

> Vswitch also has an option for this

That works, thank you!

Sorry, still wrapping my head around the naming of the plugins and stuff, thanks for your help :)
